rest/src/main/java/at/jku/isse/ecco/rest/authorisation/User.java

```diff
@@ -0,0 +1,31 @@
+package at.jku.isse.ecco.rest.authorisation;
+
+import java.util.Collection;
+
+/** Dummy User
+ * replace by actual user from preexisting DB
+ */
+
+public class User {
```

Comment:
```suggestion
public class User {
```
Pojo probably doesn't need replacing